template.yaml

```diff
@@ -358,7 +358,7 @@ Resources:
         - - "SungardAS-Alerts-Permission"
           - !Ref CloudWatchLogDestinationName
       Handler: index_account_permission.handler
-      Runtime: nodejs4.3
+      Runtime: nodejs8.10
```

Comment:
Make changes to accept this as a param

Reply:
Runtime value cannot be a Parameter. still if we need to try this , then we can do while we move code to BitBucket.